src/Utils/Image.php

```diff
@@ -280,6 +286,8 @@ public static function typeToExtension(int $type): string
 
 	/**
 	 * Returns the `Image::XXX` constant for given file extension.
+	 * @param value-of<self::Formats>|'jpg' $extension
+	 * @return self::JPEG|self::PNG|self::GIF|self::WEBP|self::AVIF|self::BMP
```

Comment:
What is `@param` good for?

Reply:
Analysis reports if unsupported format is given. Instead of finding out later when `Unsupported file extension` is thrown

Reply:
Can you think of a real example where static analysis could detect what is being sent to this method and prevent exception?

Reply:
Can you think of a real example where static analysis could detect what is being sent to this method and prevent exception?

Reply:
Yeah, probably not. Unless someone has static list of supported extensions in app, it is gonna be just annoying.
But I would keep return type, narowing down return types does no harm

Reply:
Yeah, return type is totally fine

Reply:
Good point, this would probably report more false positives than actual bugs. I'll remove it.